lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -82,10 +82,10 @@ class TraceOfTab {
     });
 
     // Find the inspected frame
-    const {startedInPageEvt, frameId} = TracingProcessor.findTracingStartedEvt(keyEvents);
+    const tracingIds = TracingProcessor.findTracingIds(keyEvents);
```

Comment:
I'm not super in love with `tracingIds` though I suppose it is a pretty accurate description of what they are. only other things I'm thinking of are `mainThreadIdentifiers` or `mainFrameIdentifiers` since that's the heart of what we're doing with them

Reply:
yeah, I just couldn't come up with a good name with so little context. maybe one day I can name things as good as the best. I'll defer to you - which do you like better?

Reply:
😆 we can only dream of being close to the naming master @paulirish 

since we've got `frame` in the mix, I'd go with `TracingProcessor.findMainFrameIds` and `mainFrameIds`, but others might have better ideas